build/guide/multi-platform.md

```diff
@@ -0,0 +1,263 @@
+builds. So first, you need to switch to a different builder, that uses a driver
+which supports concurrent multi-platform builds.
+
+To switch to using a different driver, you're going to need to use the Docker
+Buildx. Buildx is the next generation build client, and it provides a similar
+user experience to the regular `docker build` command that you’re used to, while
+supporting additional features.
+
+## Buildx setup
+
+Buildx comes pre-bundled with Docker Desktop, and you can invoke this build
+client using the `docker buildx` command. No need for any additional setup. If
+you installed Docker Engine manually, you may need to install the Buildx plugin
+separately. See
+[Install Docker Buildx](https://docs.docker.com/build/buildx/install/) for
+instructions.
+
+Verify that the Buildx client is installed on your system, and that you’re able
+to run it:
+
+```console
+$ docker buildx version
+github.com/docker/buildx v0.10.3 79e156beb11f697f06ac67fa1fb958e4762c0fab
+```
+
+Next, create a builder that uses the `docker-container`. Run the following
+`docker buildx create` command:
+
+```console
+$ docker buildx create --driver=docker-container --name=container
+```
+
+This creates a new builder with the name `container`. You can list available
+builders with `docker buildx ls`.
+
+```console
+$ docker buildx ls
+NAME/NODE           DRIVER/ENDPOINT               STATUS
+container           docker-container
+  container_0       unix:///var/run/docker.sock   inactive
+default *           docker
+  default           default                       running
+desktop-linux       docker
+  desktop-linux     desktop-linux                 running
+```
+
+The status for the new `container` builder is inactive. That's fine - it's
+because you haven't started using it yet.
+
+## Build using emulation
+
+To run multi-platform builds with Buildx, invoke the `docker buildx build`
+command, and pass it the same arguments as you did to the regular `docker build`
+command before. Only this time, also add:
+
+- `--builder=container` to select the new builder
+- `--platform=linux/amd64,linux/arm/v7,linux/arm64/v8` to build for multiple
+  architectures at once
+
+```console
+$ docker buildx build \
+    --target=binaries \
+    --output=bin \
+    --builder=container \
+    --platform=linux/amd64,linux/arm64,linux/arm/v7 .
+```
+
+This command uses emulation to run the same build four times, once for each
+platform. The build results are exported to a `bin` directory.
+
+```text
+bin
+├── linux_amd64
+│   ├── client
+│   └── server
+├── linux_arm64
+│   ├── client
+│   └── server
+└── linux_arm_v7
+    ├── client
+    └── server
+```
+
+When you build using a builder that supports multi-platform builds, the builder
+runs all of the build steps under emulation for each platform that you specify.
+Effectively forking the build into two concurrent processes.
+
+![Build pipelines using emulation](./images/emulated-builds.png){:.invertible}
```

Comment:
Ah I love this diagram!

I think we should probably emphasise that only the non-native architectures are emulated, the one that matches your current arch won't be emulated.

Reply:
Yes agree to highlight the default platform if possible. Maybe like tonis have done in https://medium.com/@tonistiigi/faster-multi-platform-builds-dockerfile-cross-compilation-guide-part-1-ec087c719eaf